tests/examples/assets/dashAgGridFunctions.js

```diff
@@ -55,4 +55,85 @@ dagfuncs.startWith = ([filterValues], cellValue) => {
 
 dagfuncs.quickFilterMatcher = (quickFilterParts, rowQuickFilterAggregateText) => {
     return quickFilterParts.every(part => rowQuickFilterAggregateText.match(part));
-}
\ No newline at end of file
+}
+
+dagfuncs.valueParser = (params) => {
+    return params.newValue != null && params.newValue.match('\\d{2}/\\d{2}/\\d{4}')
```

Comment:
```suggestion
    return params.newValue != null && params.newValue.match(/\d{2}\/\d{2}\/\d{4}/)
```
Here you're in plain JS so the usual JS regex format should work fine.